docs/CHANGES_4.0.0.md

```diff
@@ -1,39 +1,256 @@
+- [`NODE-2602`](https://jira.mongodb.org/browse/NODE-2602): createIndexOp returns string, CreateIndexesOp returns array ([#2666](https://github.com/mongodb/node-mongodb-native/pull/2666))
+- [`NODE-2936`](https://jira.mongodb.org/browse/NODE-2936): conform CRUD result types to specification ([#2651](https://github.com/mongodb/node-mongodb-native/pull/2651))
+- [`NODE-1487`](https://jira.mongodb.org/browse/NODE-1487): remove deprecated Collection.group helper ([#2609](https://github.com/mongodb/node-mongodb-native/pull/2609))
+- [`NODE-2590`](https://jira.mongodb.org/browse/NODE-2590): adds async iterator for custom promises ([#2578](https://github.com/mongodb/node-mongodb-native/pull/2578))
+- [`NODE-2458`](https://jira.mongodb.org/browse/NODE-2458): format sort in cursor and in sort builder ([#2573](https://github.com/mongodb/node-mongodb-native/pull/2573))
+- [`NODE-2324`](https://jira.mongodb.org/browse/NODE-2324): remove Cursor#transformStream ([#2574](https://github.com/mongodb/node-mongodb-native/pull/2574))
+- [`NODE-2816`](https://jira.mongodb.org/browse/NODE-2816): remove deprecated find options ([#2571](https://github.com/mongodb/node-mongodb-native/pull/2571))
+- [`NODE-2820`](https://jira.mongodb.org/browse/NODE-2820): pull CursorStream out of Cursor ([#2543](https://github.com/mongodb/node-mongodb-native/pull/2543))
+- [`NODE-2320`](https://jira.mongodb.org/browse/NODE-2320): remove deprecated GridFS API ([#2290](https://github.com/mongodb/node-mongodb-native/pull/2290))
+- [`NODE-2850`](https://jira.mongodb.org/browse/NODE-2850): only store topology on MongoClient  ([#2594](https://github.com/mongodb/node-mongodb-native/pull/2594))
+
+## Removed deprecations
+
+- `Collection.prototype.find / findOne options:`
+  - `fields` - use `projection` instead
+- `Collection.prototype.save` - use `insertOne` instead
+- `Collection.prototype.dropAllIndexes`
+- `Collection.prototype.ensureIndex`
+- `Collection.prototype.findAndModify` - use `findOneAndUpdate`/`findOneAndReplace`
+- `Collection.prototype.findAndRemove` - use `findOneAndDelete` instead
+- `Collection.prototype.parallelCollectionScan`
+- `MongoError.create`
+- `Topology.destroy`
+- `Cursor.prototype.each` - `forEach`
+- `Db.prototype.eval`
+- `Db.prototype.ensureIndex`
+- `Db.prototype.profilingInfo`
+- `MongoClient.prototype.logout`
+- `MongoClient.prototype.addUser: Creating a user without roles`
+- `MongoClient.prototype.connect`
+- `Remove MongoClient.isConnected` - calling connect is a no-op if already connected
+- `Remove MongoClient.logOut`
+- `require('mongodb').instrument`
+  - Use command monitoring: `client.on('commandStarted', (ev) => {})`
+- Top-Level export no longer a function: `typeof require('mongodb') !== 'function'`
+  - Must construct a MongoClient and call `.connect()` on it.
+- No more `Symbol` export, now `BSONSymbol` which is a deprecated BSON type
```

Comment:
Wrote some text, looks good?

Reply:
maybe change the order and make one sentence?
"Existing bson symbols in your database will be deserialized to a BSONSymbol instance; however, users should use plain strings instead of BSONSymbol"

Reply:
SGTM, done!